pkg/k8s/resource/resource.go

```diff
@@ -89,11 +89,23 @@ func FetchKubernetesResources(ctx context.Context, k *k8s.KubernetesAPI, options
 	}
 	resources = append(resources, clusterRoleBindings...)
 
-	roleBindings, err := fetchKubeSystemRoleBindings(ctx, k, options)
+	kubeSystemRoleBindings, err := fetchKubeSystemRoleBindings(ctx, k, options)
 	if err != nil {
 		return nil, fmt.Errorf("could not fetch RoleBindings from kube-system namespace:%v", err)
 	}
-	resources = append(resources, roleBindings...)
+	resources = append(resources, kubeSystemRoleBindings...)
+
+	roles, err := fetchControlNamespaceRoles(ctx, k, options, namespace)
+	if err != nil {
+		return nil, fmt.Errorf("could not fetch Roles from %s namespace:%v", namespace, err)
+	}
+	resources = append(resources, roles...)
+
+	namespaceRoleBindings, err := fetchControlNamespaceRoleBindings(ctx, k, options, namespace)
```

Comment:
nit: wdyt about keeping this as roleBindings to go with the `roles` name on line 98?